core/templates/domain/topic/topic-creation-backend-api.service.ts

```diff
@@ -56,9 +58,9 @@ export class TopicCreationBackendApiService {
   }
 
   createTopic(
-      topicName: string, abbreviatedTopicName: string): PromiseLike<Object> {
+      topicObject): PromiseLike<Object> {
```

Comment:
Every argument needs a typedef (here and elsewhere).

Also, the previous pattern in the code was good. Either have a domain object or split the variables out into separate args.

Reply:
Done.